plugins/modules/ipadnsrecord.py

```diff
@@ -1408,6 +1408,13 @@ def define_commands_for_absent_state(module, zone_name, entry, res_find):
     return _commands
 
 
+# pylint: disable=unused-argument
+def exception_handler(module, ex):
+    if ex in [ipalib_errors.EmptyModlist, ipalib_errors.DuplicateEntry]:
```

Comment:
Would it be better to test for the instance type? Like `if isinstance(ex, (ipalib_errors.EmptyModlist, ipalib_errors.DuplicateEntry])):`.

Reply:
Fixed